components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -134,7 +157,20 @@ typedef NS_ENUM(NSInteger, MDCTypographySchemeDefaults) {
 @property(nonatomic, nonnull, readwrite, copy) UIFont *caption;
 @property(nonatomic, nonnull, readwrite, copy) UIFont *button;
 @property(nonatomic, nonnull, readwrite, copy) UIFont *overline;
-@property(nonatomic, readwrite) BOOL mdc_adjustsFontForContentSizeCategory;
+
+/**
+ A hint for how fonts in this scheme should be applied to components in relation to Dynamic Type.
+
+ @note Enabling this flag only has an effect if the fonts stored on this scheme are scalable. See
+ MDCTypographySchemeDefaults for default versions that are scalable. Alternatively, you can specify
+ custom scalable fonts using the MDCFontScaler API.
+
+ When fonts are applied to components:
+
+ - If this flag is disabled, make no changes to the font.
+ - If this flag is enabled, adjust the font with respect to the current content size category.
+ */
+@property(nonatomic, readwrite) BOOL useCurrentContentSizeCategoryWhenApplied;
```

Comment:
_Tip:_ I believe this file uses explicit property annotations, maybe beneficially to add `assign`.